frontend/src/features/upload/components/AddFileItem.tsx

```
@@ -0,0 +1,21 @@
+import { UploadPageIcons as I } from '@/features/upload/icons';
+
+const AddFileItem = () => {
+  return (
+    <button className='w-[25.5rem] h-[23.5rem] flex flex-col justify-center items-center rounded-lg bg-gray-50 gap-5 cursor-pointer'>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)